### DISCUSSION TITLE
Option in Entries field settings to make it relate ancestors

### DISCUSSION THREAD

**Discussion by carlcs** [craftcms/cms#8522] [Ideas] [answered]
Basically as a first step for https://github.com/craftcms/cms/issues/1949, “Make Categories = Structures”.

**Comment by brandonkelly** [comment, marked as answer]
We’ve made this possible for Craft 4.4 🎉 (#11088)

Entries and Categories fields now have “Maintain hierarchy” settings, which are available when a single structured source is selected. When enabled, the field will behave like Categories fields have in the past (automatically relateing ancestors, and ordering relations based on the structure).

**Reply by brandonkelly** [reply]
Craft 4.4 is out with those changes!
